composer.json

```diff
@@ -33,7 +33,8 @@
     "mockery/mockery": "~1.0",
     "phpunit/phpunit": "^9.5",
     "squizlabs/php_codesniffer": "^3.5.2",
-    "scrutinizer/ocular": "^1.8.1"
+    "scrutinizer/ocular": "^1.8.1",
+    "symfony/console": "~5.0"
```

Comment:
Hebben we Symfony console nodig? Of is dit de fix?

Reply:
Dit is de fix. crutinizer/ocular wilt console hebben tussen v2 en v6. Alleen is v6 hier het probleem.